js/index.js

```diff
@@ -2,22 +2,31 @@
 
 function updateSubtotal(product) {
```

Comment:
A bit of vertical spacing would improve the readability. For example like this:

` let price = 0;
  let quantity = 0;

  price = product.querySelector('.price span').innerText;
  quantity = product.querySelector('.quantity input').value;

  let subtotal = price * quantity;

  product.querySelector('.subtotal span').innerText = subtotal;

  return subtotal;`